src/components/Panel/Panel.tsx

```diff
@@ -13,29 +16,66 @@ import { Title } from "./Title";
  * @see `PanelContext`
  */
 export function Panel() {
-  const { content, panelOpen, closePanel } = usePanelContext();
+  const { query, panelOpen, closePanel } = usePanelContext();
 
-  function close() {
-    void closePanel();
-  }
+  const close = useCallback(() => {
+    closePanel();
+  }, [closePanel]);
 
-  const props = content
+  const props = query
     ? {
-        query: content,
-        ...content,
+        query,
+        ...query,
         close,
+        isLoading: false,
       }
     : undefined;
 
   return (
     <PanelBase panelOpen={panelOpen} closePanel={close}>
-      {!!props && (
+      {props ? (
         <>
           <Title {...props} />
           <Actions {...props} />
           <InfoIcons {...props} />
           <Details {...props} />
         </>
+      ) : (
+        <>
+          <Title isLoading={true} close={close} />
```

Comment:
this is the loading version of the panel. doesn't receive data or do anything else